src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days >= 3 && days <= 6) {
+    return (totalCost = totalCost - 20);
+  } else if (days >= 7) {
+    return (totalCost = totalCost - 50);
```

Comment:
Similar to the previous comment, avoid reassigning `totalCost` and simply return `totalCost - 50`.